src/page-content/content-index.tsx

```diff
@@ -28,15 +28,14 @@ export const hero: HeroProps = {
   id: 'index.hero',
   title: (
     <>
-      <span className={'is-themed-primary'}>Cloud native </span> identity
+      <span className={'is-themed-primary'}>Limitless</span> identities
     </>
   ),
   description:
-    'Ory is the only identity platform that can scale indefinitely and is based entirely on open source.',
-
+    'Ory provides free and open identity infrastructure, securing millions of users worldwide.',
   buttons: [
     {
-      ctaTitle: 'Start Building',
+      ctaTitle: 'Start Building. No Credit Card Required.',
```

Comment:
```suggestion
      ctaTitle: 'Start Building',
```

this needs to be changed as described in feedback
![Screenshot 2021-12-02 at 15 36 34](https://user-images.githubusercontent.com/24988314/144442962-ca9e17b8-8f71-42eb-b5ac-8e57b2577204.png)